### PR TITLE
MODULE-5: Add docker

### PR DESCRIPTION
**Pull request by TheMattBarnfield**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,28 @@ Copy `.env.template` to `.env` and fill it in with the correct values.
...
+Run either of these from the project root:
+
+##### Local dev
+
+The following command will work in WSL. If you are using a different shell you'll need to replace `"$(wslpath -w $(pwd))"`
+with the absolute path to the project root.
+
+```bash
+docker build --target dev --tag todo-app:dev .
+docker run -d -p 5000:5000 -v "$(wslpath -w $(pwd))":/app todo-app:dev
```
Comment (TheMattBarnfield) [inline README.md:42]: the env file doesn't need to be included here because it's taken from the mount

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+RUN apt-get update \
+    && apt-get install -y curl \
+    && curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+ENV PATH="${PATH}:/root/.poetry/bin"
+
+WORKDIR /app
+COPY . .
+
+RUN poetry install
```
Comment (hugh-emerson) [inline Dockerfile]: You probably want the `--no-dev` option.
And you actually need to add pytest, selenium etc as dev dependencies. I think you lost them when switching from pip to poetry.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+EXPOSE 5000
+
+RUN apt-get update \
+    && apt-get install -y curl \
+    && curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+ENV PATH="${PATH}:/root/.poetry/bin"
+
+WORKDIR /app
+COPY . .
```
Comment (hugh-emerson) [inline Dockerfile]: You could copy just the poetry files and then copy the rest after poetry install.

**Review by hugh-emerson — APPROVED** [review]
Looks good! Merge when you're ready; I've just left some minor comments.

**Pull request merged**